src/syntax/lexer.ml

```diff
@@ -111,6 +111,25 @@ let is_valid_identifier s =
 		with Exit ->
 			false
 
+let split_int_suffix s =
+	let is_signed = String.contains s 'i' in
+	match String.index_opt s (if is_signed then 'i' else 'u') with
```

Comment:
Wouldn't `try String.index` show better performance? I think numeric constants are quite numerous in hx sources.

Reply:
This is unlikely to ever be measurable, and it's not clear if the alternative is actually better.

Reply:
Guess the match could be changed to something like `match (try String.index s 'i' with _ -> (try String.index s 'u' with _ -> -1)) with ...` which is less readable at a glance for me, not sure if any performance difference would really be noticable to make it worthwhile